src/components/utils/NotificationContextHolder/index.tsx

```diff
@@ -15,6 +17,9 @@ const NotificationContextHolder = () => {
     if (notification) {
       antNotification.open({
         ...notification,
+        description: React.createElement('div', {
```

Comment:
could you add a comment that says that: `notification.description` is local code (as opposed to an external server)?